jenkins/shared-library/vars/defaultCiPipeline.groovy

```diff
@@ -171,6 +191,11 @@ void call(Closure body) {
 								setupTests(env.TEST_VECTORS_SCRIPT_FILEPATH)
 							}
 						}
+						post {
+							always {
+								addEmbeddableBadgeConfiguration(id: "${params.packageId}-vectors", subject: 'vectors')
```

Comment:
Just one thing, wondering if the ``addEmbeddableBadgeConfiguration`` move to the step function like ``setupTests``.  If a new template is created it should automatically report.

Reply:
not sure, since we're adding it in other (non-test) stages too